src/swap/saga.ts

```diff
@@ -285,11 +285,17 @@ export function* swapSubmitSaga(action: PayloadAction<SwapInfo>) {
       })
     )
 
-    AppAnalytics.track(SwapEvents.swap_execute_success, {
-      ...defaultSwapExecuteProps,
-      ...getTimeMetrics(),
-      ...getSwapTxsReceiptAnalyticsProperties(trackedTxs, networkId, tokensById),
-    })
+    if (swapType === 'same-chain') {
```

Comment:
i wonder if we want to extend this block to also include the dispatching the `swapSuccess` action and even waiting for the txReceipt?